packages/utils/src/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from './promise';
+export * from './array';
+export * from './file';
+export * from './object';
+export * from './random';
+export * from './string';
+export * from './validators';
```

Comment:
I am not sure myself whether we should export like this
```
export * from './validators';
```

or like this
````
export * as validators from './validators'
````
I am still not sure which one is better :thinking:

Reply:
definitely first one, no need to add another layer of wrapping, package itself is enough

Reply:
it's mostly just some helper functions that could be used anywhere, not only as validators imho, also if we want to wrap them we should not do that here but in that file, to keep everything in one place and to not hide it in exports

Reply:
ok, let's leave it as it is now.